### PR TITLE
docs: fix typo in README

### PR DESCRIPTION
**Pull request by zyqhpz**
- fix the typo from "everthing" to "everything" in README

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

**Review by lenny-intel — APPROVED** [review]

**Comment by cloudxxx8** [comment]
the commit message should follow the syntax
please update the commit message and PR title to:
`docs: fix typo in README`

**Comment by cloudxxx8** [comment]
can you squash the commit to one, the Semantic PR check will always be failed?
all the commit message should follow the Semantic PR convention

File: CHANGELOG.md
Comment (cloudxxx8) [inline CHANGELOG.md:1]: this file change should not be included in this PR

Comment (zyqhpz) [reply CHANGELOG.md:1]: I have excluded the file

**Review by cloudxxx8 — APPROVED** [review]
LGTM
